src/Nova/Role.php

```diff
@@ -115,15 +130,23 @@ public static function singularLabel()
     }
 
     /**
-     * Load all permissions
+     * Load all permissions and cache for 1 minute.
+     * Enough to avoid N+1 at the Role index page,
+     * and not long enough to have them stalled.
      *
      * @return \Illuminate\Database\Eloquent\Collection
      */
     protected function loadPermissions()
     {
-        /** @var class-string */
-        $permissionClass = config('permission.models.permission');
-
-        return $permissionClass::all()->unique('name');
+        return cache()->remember(
+            'sereny-all-permissions',
```

Comment:
Use a const for `'sereny-all-permissions'` instead?

Reply:
Done!